docs/sections/use-case-requirements.html

```diff
@@ -10,30 +10,230 @@ <h3>Use Cases and Requirements</h3>
 
   <section>
     <h4>Identifier Discovery</h4>
-    <dl class="termlist">
-      <p>
-        As supply chains invest in web compatible standards such as
-        <a href="https://www.gs1.org/standards/gs1-digital-link"
-        >GS1 Digital Link</a
-        >
-        and <a href="https://www.gs1.org/standards/epcis">GS1 EPCIS</a>, the
-        need to discover identifiers associated with legal entities a products
-        and associated web origins becomes critical to the adoption of
-        verifiable credentials in a supply chain context.
-      </p>
 
-      <p>
-        This API specification describes how existing web standards can be
-        applied to the identifier discovery use case for supply chain actors and
-        products.
-      </p>
-    </dl>
-    <p>Endpoints of interest:</p>
-    <ul>
-      <li>
-        <a href="#api-spec">Well Known DID Configuration</a>
-      </li>
-    </ul>
+<p>
+As supply chains invest in web compatible standards such as
+<a href="https://www.gs1.org/standards/gs1-digital-link"
+>GS1 Digital Link</a
+>
+and <a href="https://www.gs1.org/standards/epcis">GS1 EPCIS</a>, the
+need to discover identifiers associated with legal entities, products
+and associated web origins becomes critical to the adoption of
+verifiable credentials in a supply chain context.
+</p>
+
+<p>
+This API specification describes how existing web standards can be
+applied to the identifier discovery use case for supply chain actors and
+products.
+</p>
+
+<p class="advisement">
+The <a href="https://github.com/w3c-ccg/did-method-web">did:web</a>
+method MUST be supported.
+</p>
+
+<p>
+The purpose of this document is to define the discoverability feature
+necessary to complete integration tests in postman with authentication.
+</p>
+
+<p>
+The current APIs exploit the well known did configuration,
+which has some limitations, most notably regarding multi-tenant platforms.
+</p>
+
+<section>
+<h5>Integration Considerations</h5>
+<p>
+An organization will have an application authorized to act on their behalf.
+</p>
+
+<p>
+The application will be identified with a <code>CLIENT_ID</code> which MAY be publically shared.
+</p>
+<p>
```

Comment:
Relocated from the end of the doc... This should also be rephrased, probably (if I understand the shorthand used here) along the lines of ``` include specific DIDs per `CLIENT_ID` in an `allow-list` ```
```suggestion
<p class="advisement">
Implementations are encouraged to `allow-list` specific DIDs per `CLIENT_ID`.
</p>

<p>
```